.markdownlint-cli2.jsonc

```diff
@@ -45,7 +45,8 @@
         "iframe",
         "details",
         "summary",
-        "pre"
+        "pre",
+        "code"
```

Comment:
We really don't use the code tag enough in the curriculum that I don't think it's worth adding to this list. The git basics lesson is really the only lesson we need to use it (the other 2 lessons should replace the tag with backticks).

So let's remove this update... (see my other comment below)